crates/cli/src/sarif/sarif_utils.rs

```diff
@@ -152,6 +166,23 @@ pub struct SarifGenerationOptions {
     pub execution_time_secs: u64,
 }
 
+impl IntoSarif for &SecretRule {
+    type SarifType = sarif::ReportingDescriptor;
+
+    fn into_sarif(self) -> Self::SarifType {
+        let mut builder = sarif::ReportingDescriptorBuilder::default();
+        builder.id(&self.name);
+
+        let description = sarif::MultiformatMessageStringBuilder::default()
+            .text(std::str::from_utf8(self.description.as_bytes()).unwrap())
+            .build()
+            .unwrap();
+        builder.full_description(description);
```

Comment:
Might be nice to use `expect` instead of `unwrap` in this function, just to add some useful context/info in case this does panic (realistically, it shouldn't)